### ISSUE TITLE
Example app - updating and deleting circle annotations is not working

### ISSUE THREAD

**Issue by FirentisTFW** [mapbox/mapbox-maps-flutter#231]
**Steps:**
1. Open the example app
2. Tap on `Circle Annotations` tile
3. Tap on `delete a circle annotation` and `update a circle annotation` buttons
4. Verify the result

**Expected result:**
Selected circle annotation gets updated/deleted. I assume we can select an annotation by tapping on it and creating a new one.
**Actual result:**
No actions performed when buttons are tapped.

**Video:**

https://github.com/mapbox/mapbox-maps-flutter/assets/55327556/04efd449-f2f7-418f-bac6-efcb99f81424

**Comment by FirentisTFW** [comment]
I just created a PR that fixes it: https://github.com/mapbox/mapbox-maps-flutter/pull/232

**Comment by futbolin** [comment]
When updating the markers the same thing happens

**Comment by FirentisTFW** [comment]
@evil159 Since #232 was merged, can we close this issue now?

**Comment by O-Hannonen** [comment]
@FirentisTFW yes this issue can be closed now. Thanks for reporting and fixing this!
